<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_findBuild">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copyBuild">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Build</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_userProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>profile</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_systemCalibration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>sys cal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_deviceProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>device profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteSelected">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_findBuildResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getLatestJenkinsBuild">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Latest Jenkins 3.0 Build</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fileshareLatestPath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileshareLatestPath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fileshare path to latest build directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_installLatestJenkinsBuild">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteOldBuilds">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Old Builds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
